packages/react/src/Icons/Icon.client.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import * as Icons from "@mui/icons-material";
+import "./Icon.client.stories.css";
+
+const meta: Meta = {
+  title: "Look'n feel/Icons",
+};
+
+export default meta;
+
+export const Example: StoryObj = {
+  name: "Icon",
+  render: ({ ...args }) => <Icons.Headphones {...args} />,
+  args: {},
+};
+
+export const AllIcons: StoryObj = {
+  render: () => {
+    return (
+      <ul className="icon-list">
+        {Object.values(Icons).map((x, i) => {
+          const Comp = x as Icons.SvgIconComponent;
+          const key = Object.keys(Icons).at(i);
+          return (
+            <li key={key}>
```

Comment:
C'est pas une best Practice de lié l'index du map à la key
ce que tu peux faire c'est utiliser le muiName de l'icon material
```suggestion
          return (
            <li key={Comp.muiName}>
```